airflow/www/templates/airflow/dags.html

```diff
@@ -170,12 +173,16 @@ <h2>{{ page_title }}</h2>
                   <a></a>
                   <span aria-hidden="true" title=" " class="material-icons text-muted js-tooltip" style="display:none">info</span>
                 </td>
-                {# Column 7: Recent Tasks #}
+                {# Column 7: Next Run #}
+                <td class="text-nowrap next-run">
+                  {{ dag.next_dagrun }}
+                </td>
+                {# Column 8: Recent Tasks #}
                 <td style="padding:0; width:323px; height:10px;">
                   {{ loading_dots(classes='js-loading-task-stats text-muted') }}
                   <svg height="10" width="10" id='task-run-{{ dag.safe_dag_id }}' style="display: block;"></svg>
                 </td>
-                {# Column 8: Actions #}
+                {# Column 9: Actions #}
```

Comment:
We should probably just remove all of these comments. They're a bit superfluous and more puissance than helpful IMO.